docs/source/release-notes.html.md.erb

```diff
@@ -5,6 +5,10 @@ weight: 2
 
 # Release Notes
 
+# 1 November 2022
+
+- Add `campaign_name` field to `CourseAttributes`. This is an array of strings, corresponding to campaigns created to help improve recruitment.
```

Comment:
Typo - this is just a single string value which is returned not an array

Reply:
Thanks - updated

https://github.com/DFE-Digital/publish-teacher-training/pull/3092/commits/75b06946d94546eb10f2567da4a49f6fccb3e449